src/db/database.ts

```diff
@@ -0,0 +1,38 @@
+import { Pool } from 'pg';
+import config from 'config';
+
+type postgresConfiguration = {
+  host: string;
+  password: string;
+  user: string;
+  database: string;
+  port: number;
+};
+
+const createConnectionString = (): string => {
+  const dbPostgres: postgresConfiguration = config.get(
+    'App.postgresConfiguration'
+  );
+  const { user, password, host, port, database } = dbPostgres;
+  return `postgresql://${user}:${password}@${host}:${port}/${database}`;
```

Comment:
You're building connection string by yourself. Imo better would be passing already built connection string as env var.

Reply:
I cannot see the advantage of passing into environment variables directly connection string.
I need to pass environment variables to the postgres container. Postgres requires user, password and database name to successfully create a connection. With the way my code currently looks it is easy to maintain because both the app and database use one source of this information, the .env file.
Creating a connection directly in an environment variable will be impractical because it will force me to change the connection parameters in two places in the event of any change. In the .env and in the node-config parameters.
When you read this and decide that I should change it after all, I will. What I have written is an argumentation of why I have chosen this way of connecting to the database.